src/main/views/home.njk

```diff
@@ -1,5 +1,106 @@
 {% extends "template.njk" %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/file-upload/macro.njk" import govukFileUpload %}
+{% from "govuk/components/select/macro.njk" import govukSelect %}
+{% from "govuk/components/textarea/macro.njk" import govukTextarea %}
+
 
 {% block content %}
-  <h1 class="govuk-heading-xl">Default page template</h1>
+  <h1 class="govuk-heading-xl">Bulk scan upload</h1>
+
+  <form action="/" method="POST" enctype="multipart/form-data">
+
+    {{ govukSelect({
+      id: "environment",
+      name: "environment",
+      label: {
+        text: "Please select environment:"
+      },
+      items: [
+        {
+          value: "aat",
+          text: "aat"
+        },
+        {
+          value: "demo",
+          text: "demo",
+          selected: true
+        },
+        {
+          value: "perftest",
+          text: "perftest"
+        },
+        {
+          value: "ithc",
+          text: "ithc",
+          disabled: true
+        }
+      ]
+    }) }}
+
+    {{ govukSelect({
+      id: "jurisdiction",
+      name: "jurisdiction",
+      label: {
+        text: "Please select jurisdiction:"
+      },
+      items: [
+        {
+          value: "probate",
+          text: "probate",
+          selected: true
+        },
+        {
+          value: "sscs",
+          text: "sscs"
+        },
+        {
+          value: "divorce",
+          text: "divorce"
+        },
+        {
+          value: "cmc",
+          text: "cmc"
+        },
+        {
+          value: "privatelaw",
+          text: "privatelaw"
+        },
+        {
+          value: "publiclaw",
+          text: "publiclaw"
+        },
+        {
+          value: "nfd",
+          text: "nfd"
+        },
+        {
+          value: "finrem",
+          text: "finrem"
+        }
+      ]
+    }) }}
+
+    <input type="file" name="myfile" />
+
+    <span class="spacer"></span>
+
+    <button type="submit" class="govuk-button" data-module="govuk-button">
+      Submit
```

Comment:
Could use the nunjucks govuk button https://design-system.service.gov.uk/components/button/

Reply:
changed